raiden/transfer/mediated_transfer/target.py

```diff
@@ -241,7 +241,7 @@ def handle_block(
         channel_state.partner_state,
         lock.secrethash,
     )
-    lock_has_expired, lock_expired_msg = channel.is_lock_expired(
+    lock_has_expired, _ = channel.is_lock_expired(
```

Comment:
Wouldn't be handy, debugging wise if we get to see the expiration reason?

Reply:
It's more like why has the lock not expired message. If it has expired we know why:

- The secret was not registered on-chain in time.
- The current block exceeds lock's expiration + confirmation blocks.

We use the debugging message of why has the lock not expired [here](https://github.com/raiden-network/raiden/blob/015fe5172e58ed1cdb723b747c346d4fb3f78346/raiden/transfer/channel.py#L547) for example